src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HPackDecoder.cs

```diff
@@ -27,6 +27,7 @@ private enum State
 
         public const int DefaultHeaderTableSize = 4096;
         public const int DefaultStringOctetsSize = 4096;
+        public const int DefaultMaxResponseHeadersLength = 65536;
```

Comment:
We should use the value from here instead of re-defining it:
https://github.com/dotnet/corefx/blob/42ecf717478504fd7cc072f3586c6edd1ba57daa/src/Common/src/System/Net/Http/HttpHandlerDefaults.cs#L19

or refactor the other place in SocketsHttpHandler that uses it and then share the constant.

Reply:
We consume it here in HttpConnectionSettings:
https://github.com/dotnet/corefx/blob/42ecf717478504fd7cc072f3586c6edd1ba57daa/src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionSettings.cs#L36

Perhaps you can use the HttpHandlerDefaults.DefaultMaxResponseHeadersLength and define it as:

```c#
public const int DefaultMaxResponseHeadersLength = HttpHandlerDefaults.DefaultMaxResponseHeadersLength * 1024;  // convert from KB to bytes
```